R/rent_assistance.R

```diff
@@ -189,6 +198,12 @@ rent_assistance <- function(fortnightly_rent = Inf,
     ra <- pminV(.prop_rent_paid_by_RA * pmaxC(Rent - min_rent, 0),
                 max_rate)
   }
+  #sharers provision
+  if (sharers_provision_applies & !is_homeowner & !has_partner & (n_dependants == 0) & lives_in_sharehouse) {
+    ra <- ra * 2/3
+  }
+  
```

Comment:
This section adjusts for those in sharehouses. I think its a bit cumbersome. The only other way I could think of was to just remove the section `!is_homeowner & !has_partner & (n_dependants == 0) & lives_in_sharehouse` and simply apply the reduction if `sharers_provision_applies == TRUE`